carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/Screenshot.java

```diff
@@ -305,7 +313,7 @@ private static String capture(WebDriver driver, boolean isTakeScreenshot, String
         				
         				augmentedDriver = driver = (WebDriver) locatorField.get(innerProxy); 
     				} catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS1166)